setup.py

```diff
@@ -8,7 +8,7 @@
 from reframe import VERSION
 
 with open('README.md') as read_me:
-    long_description = ''.join(read_me.readlines()[16:])
+    long_description = ''.join(read_me.readlines()[18:])
```

Comment:
```suggestion
    long_description = ''.join(read_me.readlines()[17:])
```

Reply:
Why 17? The section starts in line 18 and 17 is a blank line. Is there a reason to have the first line blank?